services/feedz/feedz.service.js

```diff
@@ -0,0 +1,109 @@
+'use strict'
+
+const Joi = require('joi')
+const RouteBuilder = require('../route-builder')
+const { BaseJsonService, NotFound } = require('..')
+const {
+  renderVersionBadge,
+  searchServiceUrl,
+  stripBuildMetadata,
+  selectVersion,
+} = require('../nuget/nuget-helpers')
+
+const schema = Joi.object({
+  items: Joi.array()
+    .items(
+      Joi.object({
+        items: Joi.array().items(
+          Joi.object({
+            catalogEntry: Joi.object({
+              version: Joi.string().required(),
+            }).required(),
+          })
+        ),
+      }).required()
+    )
+    .max(1)
+    .default([]),
+}).required()
+
+function apiUrl({ organization, repository }) {
+  return `https://f.feedz.io/${organization}/${repository}/nuget`
+}
+
+class FeedzVersionService extends BaseJsonService {
+  static category = 'version'
+
+  static route = new RouteBuilder({ base: 'feedz' })
+    .push('(v|vpre)', 'which')
+    .push('([^/]+)', 'organization')
+    .push('([^/]+)', 'repository')
+    .push('(.+?)', 'packageName')
+    .toObject()
```

Comment:
Is there a need to use RouteBuilder here since the route itself is static, unlike in the nuget v3 family which is dynamic?

Reply:
Yep, I think that I looked too little into other services and missed that. Fixed.